### PR TITLE
Styling/profile modal

### PR DESCRIPTION
**Pull request by bramirez96**
# Description

Made changes to UI and styling. Removed submit button - form now submits automatically when 4 digits are entered. Input field for PIN now clears when you go back to profile selection. Removed excess margin on the bottom of the form, as well as better positioned the back arrow to be consistently equidistant from the corner. Also gave the error message some more space away from the PIN input.

## Type of change

Please delete options that are not relevant.

- [x] Styling
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [x] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

## Has This Been Tested

- [ ] Yes
- [ ] No
- [x] Not necessary

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

**Review by Nick-Ohman — APPROVED** [review]
Looks great!

**Review by SandraCoburn — APPROVED** [review]
Great updates.

File: src/styles/less/ProfileModal.less [hunk 1]
```diff
@@ -29,43 +29,30 @@
...
       [class$='explain'] {
+        margin-top: 1vh;
         text-align: center;
       }
 
+      p {
+        font-size: 1.5rem;
+      }
+
       button:not(.back) {
```
Comment (Schrese) [inline src/styles/less/ProfileModal.less:47]: Thanks for showing us this!

**Review by Schrese — APPROVED** [review]
Great changes

**Pull request merged**